src/Asset/EntrypointLookup.php

```diff
@@ -134,7 +134,7 @@ private function getEntriesData(): array
             throw new \InvalidArgumentException(sprintf('Could not find an "entrypoints" key in the "%s" file', $this->entrypointJsonPath));
         }
 
-        if ($this->cache) {
+        if ($this->cache && isset($cached)) {
             $this->cache->save($cached->set($this->entriesData));
         }
```

Comment:
This technically could be just `if (isset($cached)` because `$cached` is only set if `$this->cache === true` earlier in the method